lib/languages/typescript.js

```diff
@@ -62,8 +62,13 @@ TypescriptParser.prototype.get_arg_name = function(arg) {
     if(arg.indexOf(':') > -1)
         arg = arg.split(':')[0];
 
+    var argIdentifier = /public\s+|private\s+/g;
```

Comment:
I'm not sure what you intend with this, can you please give me an example for this.
But shouldn't this be:
```js
var argIdentifier = /\bpublic\s+|\bprivate\s+/g;
```
or something alink.